src/main/resources/liveTemplates/ComposeUIArch.xml

```diff
@@ -11,4 +11,14 @@
         </context>
     </template>
 
+    <template
+        name="compara" value="@androidx.compose.ui.tooling.preview.PreviewParameterProvider&#10;class $NAME$StatePreviewParameterProvider : PreviewParameterProvider&#60;$NAME$State&#62; {&#10; override val values: Sequence&#60;$NAME$State&#62;&#10;get() = sequenceOf(&#10;&#10;)&#10;}"
```

Comment:
Not sure how I feel about the shortcut 😅 
What do you think about something like "ppp"? I think it would be shorter, tiny bit reflects whats gonna happen

Reply:
https://github.com/levinzonr/jetpack-compose-ui-arch-plugin/pull/20/commits/b48d158816e3cfab66251db7c6afe65519138872

Actually, I really don't know what the shortcut name will be, so I just wrote it as it came to my mind.
But as you said, I thought "ppp" could be really simple and easily imprinted on people.